src/components/choice/ChoiceCreation.vue

```diff
@@ -1,14 +1,25 @@
 <template>
```

Comment:
Where is menu link to creator?

Reply:
It will be implemented, when I will finish working on jwt handling, because i want to allow only logged users to access creator.